Modules/_cursesmodule.c

```diff
@@ -3348,9 +3347,7 @@ _curses_initscr_impl(PyObject *module)
     SetDictInt("LINES", LINES);
     SetDictInt("COLS", COLS);
 
-    winobj = (PyCursesWindowObject *)PyCursesWindow_New(win, NULL);
-    screen_encoding = winobj->encoding;
-    return (PyObject *)winobj;
+    return (PyObject *)PyCursesWindow_New(win, screen_encoding);
```

Comment:
This is not a nonfunctional change, as described in the topic.

Reply:
You're right, I messed this up a little :smile:
Still needs NULL check for `winobj` value before assign to `screen_encoding` though.

Reply:
Reverted this change as I want to fix this in #104732.